kubetest/objects/ingress.py

```diff
@@ -0,0 +1,111 @@
+"""Kubetest wrapper for the Kubernetes ``PersistentVolumeClaim`` API Object."""
+
+import logging
+
+from kubernetes import client
+
+from .api_object import ApiObject
+
+log = logging.getLogger('kubetest')
+
+
+class Ingress(ApiObject):
+    """Kubetest wrapper around a Kubernetes `Ingress`_ API Object.
+
+    The actual ``kubernetes.client.ExtensionsV1beta1Api`` instance that this
+    wraps can be accessed via the ``obj`` instance member.
+
+    This wrapper provides some convenient functionality around the
+    API Object and provides some state management for the `Ingress`_.
+
+    .. _Ingress:
+        https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#ingress-v1beta1-extensions
+    """
+
+    obj_type = client.ExtensionsV1beta1Api
+
+    api_clients = {
+        'preferred': client.ExtensionsV1beta1Api,
+        'v1': client.ExtensionsV1beta1Api,
```

Comment:
I think the key here should be `'extensions/v1beta1'` instead of `'v1'` since, IIRC, ingress is still in  beta and hasn't landed   in the stable channel  yet